flyteplugins/go/tasks/plugins/k8s/ray/batchscheduler/config/config.go

```diff
@@ -0,0 +1,21 @@
+package config
+
+type Config struct {
+	Scheduler  string `json:"scheduler,omitempty"`
+	Parameters string `json:"parameters,omitempty"`
```

Comment:
Could we add some comments or pflag to this config? here is an [example](https://github.com/flyteorg/flyte/blob/ba88e82a58145fe3062c7441cdd5c57c36f108f2/flytectl/cmd/config/config.go#L21-L24)